### PR TITLE
Raw Vpc schema integration (1.0.0 parquet )

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
update the vpc flow default integration to match the 1.0 parquet schema as shown [here](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-athena.html) 

See [release page](https://github.com/opensearch-project/opensearch-catalog/releases/tag/amazon_vpc_flow_1.0.0) in the catalog repo

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/aws_vpc_flow_flint-pre_agg-1.0.0.ndjson
Comment (ps48) [inline server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/aws_vpc_flow_flint-pre_agg-1.0.0.ndjson:1]: Looks like we are reverting all Flint multiscale aggregation changes for VPC.  Is this expected?

**Review by Swiddis — APPROVED** [review]

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/aws_vpc_flow_flint-pre_agg-1.0.0.ndjson
Comment (YANG-DB) [reply server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/aws_vpc_flow_flint-pre_agg-1.0.0.ndjson:1]: yes - this has moved to a future release, currently the aggregative queries are shown in the example queries

File: server/adaptors/integrations/__data__/repository/amazon_vpc_flow/amazon_vpc_flow-1.0.0.json [hunk 1]
```diff
@@ -110,75 +98,12 @@
...
-      "name": "aws_vpc_flow_flint-pre_agg",
-      "version": "1.0.0",
-      "extension": "ndjson",
-      "type": "savedObjectBundle",
-      "workflows": ["flint-pre-agg-dashboards"]
-    },
-    {
-      "name": "vpc_live_all_mv",
+     {
+      "name": "aws_vpc_live_stream_mv_schema",
```
Comment (RyanL1997) [inline server/adaptors/integrations/__data__/repository/amazon_vpc_flow/amazon_vpc_flow-1.0.0.json:102]: Is this name format only for VPC? Should we also apply this on all the other integrations?

**Review by RyanL1997 — APPROVED** [review]
Hi @YANG-DB , LGTM, just left a minor comment on the naming.

**Review by ps48 — APPROVED** [review]

**Pull request merged**